microsoft/surface/common/kernel/default.nix

```diff
@@ -6,6 +6,7 @@ let
 in {
   imports = [
     ./linux-6.10.x
```

Comment:
There's often no point in keeping the older kernels around, unless there's some issues with the newer kernel on certain types of hardware.

Reply:
Dropped 🗑️ Maybe if the linux-surface team releases an LTS kernel it'll be worth re-adding, but for now, we can just update one.